multipass/reference/multipass-mount-command.md

```diff
@@ -6,7 +6,12 @@ Please don't copy it blindly, first update the URL passed to the form with the c
 
 > See also: [Mount](/t/28470), [How to share data with an instance](/t/27189), [`umount`](/t/27214), [ID mapping](/t/45986).
 
-The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986).
+The `multipass mount` command maps a local directory from the host to an instance, with the possibility to specify the [mount](/t/28470) type (classic or native) and define group or user [ID mappings](/t/45986). Here's the syntax for mapping a local directory to your virtual machine using the classic mount:
+
+```shell
+multipass mount --type=classic /host/path instance:/instance/path
```

Comment:
In this command, I'd make it clearer that `instance` is a variable parameter indicating the instance name:

```suggestion
multipass mount --type=classic /host/path <instance-name>:/instance/path
```

Another possibility would be using the same instance name that is used in the related how-to, i.e. `keen-yak`, and highlighting this in the paragraph introducing the example command.

Reply:
Yes, `keen-yak` is a sample instance name, similar to `foo-bar` in a sense. 

To make it absolutely clear that this is a variable name that should be substituted with an actual instance name, I'd suggest either generalising the example completely, or using a specific, real-life example with credible paths (e.g. $HOME instead of /host/path and /mnt/home for /instance/path, or any other example that makes sense). 

I'll leave this choice to you :)

Reply:
Alright. I think the first suggestion works fine!